lib/Stats.js

```diff
@@ -460,20 +471,29 @@ Stats.jsonToString = function jsonToString(obj, useColors) {
 		colors.bold(obj.publicPath);
 		newline();
 	}
+
 	if(obj.assets && obj.assets.length > 0) {
 		var t = [
-			["Asset", "Size", "Chunks", "", "Chunk Names"]
+			[
+				{value: "Asset", color: colors.bold},
+				{value: "Size", color: colors.bold},
+				{value: "Chunks", color: colors.bold},
+				{value: "", color: colors.bold},
+				{value: "", color: colors.bold},
```

Comment:
Had to also set the default colSizes[] for empty string col but tests show fixed 👍